mappings/net/minecraft/world/storage/SerializingRegionBasedStorage.mapping

```diff
@@ -24,5 +30,11 @@ CLASS net/minecraft/class_4180 net/minecraft/world/storage/SerializingRegionBase
 		ARG 1 pos
 	METHOD method_19295 getOrCreate (J)Ljava/lang/Object;
 		ARG 1 pos
+	METHOD method_20368 update (Lnet/minecraft/class_1923;Lcom/mojang/datafixers/types/DynamicOps;Ljava/lang/Object;)V
+		ARG 1 pos
+		ARG 2 ops
+		ARG 3 data
 	METHOD method_20369 getDataVersion (Lcom/mojang/datafixers/Dynamic;)I
 	METHOD method_20370 save (Lnet/minecraft/class_1923;)V
+	METHOD method_20621 loadNbt (Lnet/minecraft/class_1923;)Lnet/minecraft/class_2487;
```

Comment:
`loadNbt`? Did you turn into Prospector?

Reply:
I recall there is a specific reason I named this, let me look into it real quick (like this is not copying wholesale but some sort of selective reading)

Reply:
Oh, this is not a `fromNbt` but rather getting nbt from disk (loaded by the storage io worker that is a field of it, calls its `getNbt` and wraps the call in an exception check). What name do you suggest?

Reply:
I'm fine with this name, I was just surprised as the current convention is to use "tag(s)" over "nbt"